Add Ruby book in Tamil, check_urls=free-programming-books-ta.md

kadhirash opened the pull request:
## What does this PR do?
Add Resource

## For resources
### Description 
Learning Ruby in Tamil
### Why is this valuable (or not)?
One can learn Ruby in a language they are comfortable with
### How do we know it's really free?
Authorized by the book's publishers website, and it is readily available for a free download in PDF format.
### For book lists, is it a book?
Yes
### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

eshellman commented:
this is why why prefer atomic PR's - this PR includes the not-free book from your previous PR.

kadhirash commented:
Hello @eshellman I think I found my mistake and fixed it now. I created a branch for this feat. from my other branch instead of master, thus the not-free PR was also included. I deleted the old branch and kept this branch and fixed up the errors. 

What do you mean exactly by atomic PR's?

eshellman commented:
by "atomic" I mean try to create a fresh branch for every issue or resource group addressed.
